lap/pom.xml

```diff
@@ -115,37 +72,37 @@
      		<groupId>de.tudarmstadt.ukp.dkpro.core</groupId>
      		<artifactId>de.tudarmstadt.ukp.dkpro.core.opennlp-asl</artifactId>
     	</dependency>
-    	
+
     	<!--  open NLP models (sentence breaker, tokenizer, tagger) used by OpenNLP LAPAccess instances  -->
-    	<dependency>
+		<dependency>
      		<groupId>de.tudarmstadt.ukp.dkpro.core</groupId>
      		<artifactId>de.tudarmstadt.ukp.dkpro.core.opennlp-model-tagger-en-maxent</artifactId>
-     		<version>20120616.0</version>
+     		<version>${dkpro-opennlp-version}</version>
```

Comment:
Instead of adding versioned model dependencies, the model versions should be imported from the respective component module. For details, see here: https://dkpro.github.io/dkpro-core/releases/1.8.0/docs/user-guide.html#_adding_components_as_dependencies_maven - but I have to admit, I do not know if this already worked back with DKPro Core 1.4.0.